README.md

```diff
@@ -10,9 +10,9 @@ Feel free to file issues or ask questions on our [issue tracker](http://github.c
 
```

Comment:
Why are we supporting a chat that we barely use? It would be better to steer the users to one spot--maybe create a dedicated post in developer community where users can ask questions? It will allow us to respond and organize information so that it can be easily referenced by future users.

Reply:
Some users asked for it. I suppose some like having a lower impact way of asking a question without officially "filing" something somewhere.

We can look into shutting it down.

Reply:
Let's take the conversation offline and make the change later if need be.